examples/adaptive_library/BezierCurveRotorPockets.py

```diff
@@ -0,0 +1,305 @@
+
+# %%
+# Define nodes from points and create curve using bezier
+nodes2 = np.asfortranarray(
+    [
+        xlist,
+        ylist,
+    ]
+)
+curve2 = bezier.Curve.from_nodes(nodes2)
+
+# %%
+# Create set of points for drawing the calculated bezier curve
+num_pts = 256
+s_vals = np.linspace(0.0, 1.0, num_pts)
+points2 = curve2.evaluate_multi(s_vals)
+
+# %%
+# add the points as Coordinate objects to a list
+xylist = []
+for i in range(num_pts):
+    c = Coordinate(points2[0, i], points2[1, i])
+    xylist.append(c)
+
+# %%
+# Create a list of entities from the curve points
+linetolerance = 0.01
+arctolerance = 0.01
+bez_curve_entities = return_entity_list(xylist, linetolerance, arctolerance)
+
+# %%
+# Add the new entities that make up the curve to the first Rotor Pocket region
+# Counts the number of arcs and lines
+arccount = 0
+linecount = 0
+for ent in bez_curve_entities:
+    Rotor_Pocket_regions[0].add_entity(ent)
+    if isinstance(ent, Arc):
+        arccount = arccount + 1
+    if isinstance(ent, Line):
+        linecount = linecount + 1
+
+# %%
+# Translate (move) the Rotor Pocket Region in the x-y plane to the magnet edge
+Rotor_Pocket_regions[0].translate(StartCoordinate.x, StartCoordinate.y)
+
+# %%
+# Rotate the Rotor Pocket Region to match the magnet edge
+Rotor_Pocket_regions[0].rotate(StartCoordinate, -(90 - MagnetFaceLine.angle))
+
+# %%
+# Add the magnet edge line to the Rotor Pocket region
+Rotor_Pocket_regions[0].add_entity(MagnetFaceLine)
+
+# %%
+# Check that the rotor pocket region is joined up and set the region in Motor-CAD
+if Rotor_Pocket_regions[0].is_closed():
+    mc.set_region(Rotor_Pocket_regions[0])
+
+# %%
+# Mirror the first rotor pocket region on the other half of the rotor
+# Creates a new region mirroredRegion
+mirrorlinex, mirrorliney = rt_to_xy(rotor_radius, pole_angle)
+mirrorLine = Line(Coordinate(0, 0), Coordinate(mirrorlinex, mirrorliney))
+mirroredRegion = Rotor_Pocket_regions[0].mirror(mirrorLine)
+
+# %%
+# Replace the entities in the second Rotor Pocket with those from mirroredRegion
+Rotor_Pocket_regions[1].replace(mirroredRegion)
+
+# %%
+# Check that the rotor pocket region is joined up and set the region in Motor-CAD
+if Rotor_Pocket_regions[1].is_closed():
+    mc.set_region(Rotor_Pocket_regions[1])
+
+# %%
+# .. image:: ../../images/Adaptive_Geometry_Bezier_e4a_3.png
+#     :width: 300pt
+
+# %%
+# Load in Adaptive Templates Script if required
+# ---------------------------------------------
+# When this script is run externally, the following is executed:
+#
+# * Set Geometry type to "Adaptive"
+#
+# * Load the script into the Adaptive Templates tab
+#
+# * Go to the Geometry -> Radial tab to run the Adaptive Templates Script and display the new
+#   geometry
+
+# %%
+# .. note::
+#    When running in a Jupyter Notebook, the path for the Adaptive Templates script (PY file) must
+#    be given instead of ``sys.argv[0]`` when using the ``load_adaptive_script()`` method.
```

Comment:
```suggestion
#    When running in a Jupyter Notebook, you must provide the path for the Adaptive Templates script (PY file)
#    instead of ``sys.argv[0]`` when using the ``load_adaptive_script()`` method.
```